readme.md

```diff
@@ -94,158 +94,229 @@ GitHub Enterprise is also supported. More info in the options.
+- <!-- warning-for-disallow-edits --> [Warns you when unchecking `Allow edits from maintainers`, as it’s maintainer-hostile.](https://user-images.githubusercontent.com/1402241/53151888-24101380-35ef-11e9-8d30-d6315ad97325.gif)
+- <!-- pull-request-hotkey --> Adds keyboard shortcuts to cycle through PR tabs: <kbd>g</kbd> <kbd>←</kbd> and <kbd>g</kbd> <kbd>→</kbd>, or <kbd>g</kbd> <kbd>1</kbd>, <kbd>g</kbd> <kbd>2</kbd>, <kbd>g</kbd> <kbd>3</kbd> and <kbd>g</kbd> <kbd>4</kbd>.
+- <!-- mark-files-as-viewed --> [Marks all files as "viewed" when approving a PR.](https://user-images.githubusercontent.com/11782/60969312-8740df00-a31f-11e9-8e34-d1f0c1a871aa.gif)
+- <!-- pr-branch-auto-delete --> Automatically deletes the branch right after merging a PR, if possible.
+
+<!-- Refer to style guide above. Keep this message between sections. -->
+
+### Commits
+
+- <!-- faster-pr-diff-options --> Adds a keyboard shortcut to ignore the whitespace in diffs: <kbd>d</kbd> <kbd>w</kbd>.
+- <!-- patch-diff-links --> [Adds links to `.patch` and `.diff` files in commits.](https://cloud.githubusercontent.com/assets/737065/13605562/22faa79e-e516-11e5-80db-2da6aa7965ac.png)
+- <!-- default-to-rich-diff --> [Renders the rich diff by default in SVG files’ diffs.](https://user-images.githubusercontent.com/5243867/57125552-c08a2b00-6d81-11e9-9b84-cdb535baa98e.png)
+- <!-- raw-file-link --> [Adds link to view the raw version of files in PRs and commits.](https://user-images.githubusercontent.com/1402241/56484988-b99f2500-6504-11e9-9748-c944e1070cc8.png)
+- <!-- faster-pr-diff-options --> [Adds one-click buttons to change diff style and to ignore the whitespace.](https://user-images.githubusercontent.com/1402241/54178764-d1c96080-44d1-11e9-889c-734ffd2a602d.png)
+- <!-- follow-file-renames --> 🔥 [Enhances files’ commit lists navigation to follow file renames.](https://user-images.githubusercontent.com/1402241/54799957-7306a280-4c9a-11e9-86de-b9764ed93397.png)
+- <!-- link-to-file-in-file-history --> [Adds links to the file itself in a file’s commit list.](https://user-images.githubusercontent.com/22439276/57195061-b88ddf00-6f6b-11e9-8ad9-13225d09266d.png)
+- <!-- extend-diff-expander --> [Widens the `Expand diff` button to be clickable across the screen.](https://user-images.githubusercontent.com/6978877/34470024-eee4f43e-ef20-11e7-9036-65094bd58960.PNG)
+- <!-- remove-diff-signs --> [Hides diff signs](https://user-images.githubusercontent.com/1402241/54807718-149cec80-4cb9-11e9-869c-e265863211e3.png) (since diffs are color coded already.)
+- <!-- limit-commit-title-length --> [Limits the commit title fields to 72 characters.](https://user-images.githubusercontent.com/37769974/60379478-106b3280-9a51-11e9-88b9-0e3607f214cd.gif)
+
+<!-- Refer to style guide above. Keep this message between sections. -->
+
+### Profiles
+
+- <!--  --> [Shows the followers you have in common.](https://user-images.githubusercontent.com/2906365/42009293-b1503f62-7a57-11e8-88f5-9c2fb3651a14.png)
```

Comment:
For those wondering, these comments are just so features can be easily found and removed in the readme when they're dropped/renamed/updated.

```suggestion
- <!-- show-followers-you-know --> [Shows the followers you have in common.](https://user-images.githubusercontent.com/2906365/42009293-b1503f62-7a57-11e8-88f5-9c2fb3651a14.png)
```